components/drops/CreateCalendarEventModal.vue

```diff
@@ -83,6 +88,12 @@ const addGoogleEvent = () => {
   window.open(calendarURL.toString(), '_blank')
 }
 
+const formattedDate = computed(() =>
+  props.showExactDate
+    ? format(FUTURE_DROP_DATE, 'dd/MM/yyyy')
```

Comment:
The exact date should be returned from api instead of hard coding every time.

Reply:
> What if we have several drops on different dates?

do we have in API scheduled time of drop iirc?
cc @vikiival

Reply:
Yes we do 

https://github.com/kodadot/private-workers/blob/abf0b070deae00d689469161619118dab10875f3/fxart/schema.sql#L15

Reply:
fixed , removed `FUTURE_DROP_DATE` since it's not used anymore , [f128249](https://github.com/kodadot/nft-gallery/pull/9677/commits/f12824988fe9d2c25e34f22de6b54b4dabf20df8)